.circleci/config.yml.j2

```diff
@@ -2,208 +2,25 @@ version: 2.1
 
 workflows:
   rebase-astro-main:
-    when:
-      and:
-        - equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
-        - equal: [ "every-midnight-utc", << pipeline.schedule.name >> ]
     jobs:
       - rebase-astro-main
-  certified-airflow:
```

Comment:
Where di this go? It's a very big diff for a small sounding change.

Reply:
My bad.. missed a commit, please check now